server/export_test.go

```diff
@@ -152,6 +153,7 @@ func setupApp(t *testing.T, tempDir string) (*simapp.SimApp, context.Context, *t
 			AppStateBytes:   genDoc.AppState,
 		},
 	)
+	app.SetInitialAppVersionInConsensusParams(app.NewContext(false, tmproto.Header{}), simapp.DefaultConsensusParams.Version.AppVersion)
```

Comment:
For backwards compatibility, app version should only be part of the consensus params from v2 onwards. Because of this we should only set the initial app version if the version is v2 (which the default is not)

Reply:
I pushed new commit to only init app version when value is equal or higher than 2 and only export app version when equal or higher than 2.